cookbooks/bcpc-hadoop/templates/default/hdp_hadoop-env.sh.erb

```diff
@@ -11,18 +11,18 @@ export HADOOP_OPTS="-Djava.net.preferIPv4Stack=true ${HADOOP_OPTS}"
 export HADOOP_LOG_DIR=/var/log/hadoop-hdfs
  
 # History server logs
-export HADOOP_MAPRED_LOG_DIR=/var/log/hadoop-hdfs
+export HADOOP_MAPRED_LOG_DIR=/var/log/hadoop-mapreduce
+
+# History server pid
+export HADOOP_MAPRED_PID_DIR=/var/run/hadoop-mapreduce
```

Comment:
Nit: this could have been left in the `# The directory where pid files are stored.` section